### ISSUE TITLE
Progression issue.

### ISSUE THREAD

**Issue by mcenderdragon** [mcenderdragon/Futurepack-API#872]
apparently alot of people stop using futurepack as sson as they need 2 items - becuase they dont know they need a research modul energy. instead they look for calculation modules. This is probaply because the tooltip mentions the calculation modul.

Improvement: add a notice to the researcher mentioning when no Research modul of the certain type is found on the network.

**Comment by Yirok42** [comment]
I do not know when or why I figured out to attach every Research Module to the researcher (after scanning) but since without the Network Cable I know of no other way for them to be useful. 
My earliest memory of getting blocked was not knowing that scanning Cable was what I needed to do (this was LONG ago). Perhaps the goal is for us to scan everything in order to make progress, but like right now, when (from prior playing) I know I need to visit Menelaus in order to unlock the next phase of research, nothing seems to make this clear. So many ? signs in the researcher (and until custom biomes are supported -- research is stopped)
